server/server.js

```diff
@@ -27,6 +29,25 @@ if (!isDev && cluster.isMaster) {
 } else {
   const app = express();
 
+  // Setup socket.io connection
+  const { createServer } = http;
+  const { Server } = socketio;
+  const httpServer = createServer(app);
+  const io = new Server(httpServer, {
```

Comment:
Is this needed for executing bots or is this an alternative for using express for routing?

Reply:
This is the necessary setup in order to use socket.io and therefore needed for executing bots.
It can be an alternative to express whenever we want a bidirectional communication channel between client and server and not the classic request/response.